Pages/Bovine/ABMBovine.cshtml.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using cema_ui.Models;
+using Microsoft.Extensions.Logging;
+using System.Text;
+using Newtonsoft.Json;
+using System.IO;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace cema_ui.Pages
+{
+    public class ABMBovine : PageModel
+    {
+        private readonly ILogger<login> _logger;
+        private readonly Uri _baseUrl = new("http://192.168.100.5:30024");
+        public string title { get; set; }
+        public string valid { get; set; }
+        public string succes { get; set; }
+        [BindProperty] public Models.Bovine bovine { get; set; }
+
+        public ABMBovine(ILogger<login> logger)
+        {
+            _logger = logger;
+        }
+
+        [Route("/Bovine/ABMBovine/{id}")]
```

Comment:
```suggestion
        [Route("/Bovine/ABMBovine")]
        [Route("/Bovine/ABMBovine/{id}")]
```
de esta forma tendriamos sin id funcionando, habria que hacer las excepciones correctas del sin id